speechbrain/lobes/models/transformer/DCT.py

```diff
@@ -0,0 +1,130 @@
+"""Configuration and utility classes for classes for Dynamic Chunk Training, as
```

Comment:
Why in transformer? Why not in speechbrain/utils?

Reply:
This is what I did before, but there's a nasty circular dependency because `core.py` imports something from `speechbrain.utils`, and `utils/__init__.py` pulls every `.py` file there (aaaa), and `DCT.py` needs `sb.core.Stage`. *Somehow,* no util prior to that broke and depended on the core...

The lazy workaround would be to bring the `Stage` import inside of the function, which is ugly. I guess we really need a proper fix but I'm not sure what.

I thought it'd be fine since this is largely applied to transformers, but on second thought, nothing about the concept is really specific to it...